### PR TITLE
Adding number of CPU cores

### PR DESCRIPTION
**Pull request by thardie**
Added a metric under cpu total "ncores" which records the number of
cores on this machine. This is useful if trying to divide any total
values by the number of CPU cores to get an actual percentage

### CODE REVIEW

**Comment by jaingaurav** [comment]
Thanks for the PR. I would request the same improve as I did for PR# 162. Please take a look and see what you can do.

Also I'd appreciate squashing the commits to logical chunks once your done.

File: src/collectors/nfacct/nfacct.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def get_default_config_help(self):
         config_help = (
             super(NetfilterAccountingCollector, self).get_default_config_help())
         config_help.update({
-            'bin': 'The path to the smartctl binary',
+            'bin': 'The path to the nfacct binary',
```
Comment (shortdudey123) [inline src/collectors/nfacct/nfacct.py]: Unrelated?
